types/tizen-common-web/tslint.json

```diff
@@ -3,6 +3,10 @@
     "rules":{
         "max-line-length": false,
         "no-empty-interface": false,
-        "interface-over-type-literal": false
+        "interface-over-type-literal": false,
+        "no-irregular-whitespace": false,
+        "no-misused-new": false,
+        "no-unnecessary-class": false,
+        "unified-signatures": false
```

Comment:
All these  new exceptions cannot be added. Please fix error instead.

Reply:
Hi, We could fix them almost. However, only about "no-misused-new" tag, we just want to avoid changing ["no-misused-new": false] to "true". Since It's related to user experience for our previous user who used to use it as Tizen api before (original our Tizen API). If it is not a critical issue, we suggest you to let us merge as it is.